Fabric.Terminology.SqlServer/Persistence/SqlValueSetCodeRepository.cs

```diff
@@ -183,18 +188,21 @@ private IReadOnlyCollection<IValueSetCode> QueryValueSetCodes(IEnumerable<Guid>
         {
             var defaultItemsPerPage = this.sharedContext.Settings.DefaultItemsPerPage;
             var pagingStrategy = this.pagingStrategyFactory.GetPagingStrategy<IValueSetCode>(defaultItemsPerPage);
+            var orderingStrategy = this.orderingStragetyFactory.GetValueSetCodeStrategy();
 
             pagingStrategy.EnsurePagerSettings(pagerSettings);
 
             var count = await source.CountAsync();
-            var items = await source.OrderBy(dto => dto.CodeDSC)
-                            .Skip((pagerSettings.CurrentPage - 1) * pagerSettings.ItemsPerPage)
+
+            source = orderingStrategy.SetOrdering(source, pagerSettings.AsOrderingParameters());
+
+            var items = await source.Skip((pagerSettings.CurrentPage - 1) * pagerSettings.ItemsPerPage)
                             .Take(pagerSettings.ItemsPerPage)
                             .ToListAsync();
 
             var factory = new ValueSetCodeFactory();
 
-            // TODO can't cache at this point since codeGuid is null in db.  Fixme
+            // TODO can't cache at this point since codeGuid is null in db.  Fix me
```

Comment:
Ensuring this TODO is referring to a future todo and not something you wanted to have completed in this PR